### DISCUSSION TITLE
I want universal support for a modern image format

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#220] [Wants]
A consistent problem I see is authors getting confused about image formats for the media they add to the site. Factors such as animation, alpha-channels, file size and quality just confuse most content authors when selecting a format. Worse, images make up a substantial amount of the bandwidth used for our sites. Imagine being able to reduce that data by 30%; that&#39;s the sort of gain that a more modern format could achieve. Less bandwidth, faster download, faster pages.

The formats we use are ancient. PNG was initially released 23 years ago; it&#39;s older than some of our developers. JPEG and GIF are even older. Yes, there has been some progressive  enhancement, but we&#39;re still stuck in the stone ages for image formats. WebP holds out some promise, but *still* lacks universal support. We need a modern, universal format.

<hr>

https://webwewant.fyi/wants/125/

**Comment by PPPDUD** [comment]
I agree, but I think that the image format should be a universal media format, not just a universal image format. The format should be able to represent images, uncompressed repeating video clips (with mono sound), and long talkies (long, losslessly compressed video clips with closed captions and stereo audio).
